server/mailer.js

```diff
@@ -87,6 +88,30 @@ class Mailer {
 }
 
 const mailer = new Mailer();
+const mailerQueue = new Queue('email', process.env.REDIS_URL);
+
+mailerQueue.process(async function(job) {
+  // $FlowIssue flow doesn't like dynamic values
+  await mailer[job.data.type](job.data.opts);
```

Comment:
You should type the strings that `job.data.type` can have (the individual email methods)... eg something like:

```
type Job = {
  data: {
    type: "welcome" | "password_reset"
  }
}